src/swsssdk/scripts/sonic-db-cli

```diff
@@ -7,6 +7,14 @@ import argparse
 from multiprocessing import Pool
 from functools import partial
 
+def print_response(resp):
+    if resp is None:
+        print()
+    elif isinstance(resp, list):
```

Comment:
@dzhangalibaba  Can you please comment on the below observation.

After more testing, I observed that when we give the below command, it is printing out the dictionary and not the list like in redis-cli

In py3
sonic-db-cli CONFIG_DB HGETALL "PORT|Ethernet0"
{b'index': b'0', b'alias': b'Ethernet1/0/1', b'admin_status': b'up', b'mtu': b'9100', b'speed': b'40000', b'lanes': b'1,2,3,4'}

In py2
root@sonic:/home/admin# sonic-db-cli CONFIG_DB HGETALL "PORT|Ethernet0"
{'index': '0', 'lanes': '1,2,3,4', 'mtu': '9100', 'alias': 'Ethernet1/0/1', 'admin_status': 'up', 'speed': '40000'}

There is a difference if we use hgetall
root@sonic:/home/admin# sonic-db-cli CONFIG_DB hgetall "PORT|Ethernet0"
index
0
lanes
1,2,3,4
mtu
9100
alias
Ethernet1/0/1
admin_status
up
speed
40000

Reply:
looks the redis-py has some different behavior when the input parameter are in different mformat. When quota "HGETALL PORT|Ethernet0", result is a list, otherwise is a dict . Interesting. The good thing is looks no application use HGETALL in current SONiC codes, it they want to use, need quote the whole command. Better to convert the dict to a list to avoid possible confuse. 

```
admin@ASW-7005:~$ sonic-db-cli CONFIG_DB HGETALL "PORT|Ethernet0"
{'alias': 'Ethernet1/1', 'admin_status': 'up', 'lanes': '17', 'index': '0', 'mtu': '9100'}
admin@ASW-7005:~$ sonic-db-cli  CONFIG_DB "HGETALL PORT|Ethernet0"
alias
Ethernet1/1
admin_status
up
mtu
9100
lanes
17
index
0

```

could you add something like this, then I don't need to raise another PR.
```
def print_response(resp):

    if isinstance(resp, dict):
        resp = list(sum(zip(resp.keys(),resp.values()),()))

    if resp is None:
        print()
    elif isinstance(resp, list):
        for x in resp: print(x.decode('utf-8') if isinstance(x, bytes) else x)
    else:
        print(resp.decode('utf-8') if isinstance(resp, bytes) else resp)

```

Reply:
With latest changes to use decode_responses=True in the dbconn, everything shows up right. Both hgetall and HGETALL. 

root@sonic:/home/admin# sonic-db-cli CONFIG_DB "hgetall PORT|Ethernet0"
index
0
lanes
1,2,3,4
mtu
9100
alias
Ethernet1/0/1
admin_status
up
speed
40000
root@sonic:/home/admin# sonic-db-cli CONFIG_DB "HGETALL PORT|Ethernet0"
index
0
lanes
1,2,3,4
mtu
9100
alias
Ethernet1/0/1
admin_status
up
speed
40000
root@sonic:/home/admin#

Reply:
@dzhangalibaba Reading the code of `redis/client.py` I feel we need to call `c.execute_command('hgetall', 'PORT|Ethernet0')` instead of `c.execute_command('hgetall PORT|Ethernet0')`, because `execute_command` will use `arg[0]` to post-parse response. We need to consider this and limit in `sonic-db-cli`. Could you raise another PR to fix them?